entsoe/parsers.py

```diff
@@ -483,7 +483,17 @@ def _parse_crossborder_flows_timeseries(soup):
 
     series = pd.Series(index=positions, data=flows)
     series = series.sort_index()
-    series.index = _parse_datetimeindex(soup)
+    index = _parse_datetimeindex(soup)
+
+    # quick fix for some queries that report a 15M increment,
+    # when data is really hourly
+    if len(index) != len(series):
```

Comment:
There's a weird API quirk for some BZs where they return hourly data (as expected), but self report in the API response that the data is actually 15 minute increment. This leads to an unequal shape of the index created by the parser.